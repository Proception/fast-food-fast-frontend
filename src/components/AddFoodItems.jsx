import React, { Component } from 'react';

export default class Home extends Component {
    render() {
      return (
      <section>
				<div class="container">
					<div id="wrapper">
					<h1>Enter Food Item details</h1>
						<div class="register">
							<p><input class="reg-control" placeholder="Food Name" type="text" name=""/></p>
							<p><input class="reg-control" placeholder="price" type="number" name=""/></p>
							<p><input class="reg-control" placeholder="Quantity/portions Available" type="number" name=""/></p>
							<p><textarea name=""  placeholder="Description" cols="42" rows="5"></textarea></p>
							<p><input class="reg-control" type="file" name="pic" accept="image/*"/></p>
							<p><input class="button" type="button" name="" value="Submit"/></p>
						</div>
					</div>
				</div>
			</section>
      )
    }
  }