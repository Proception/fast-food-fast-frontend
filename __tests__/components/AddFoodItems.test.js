import React from 'react';
import { shallow } from 'enzyme';
import AddFoodItems from '../../src/components/AddFoodItems.jsx';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<AddFoodItems/>);
});
describe('AddFoodItems Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
