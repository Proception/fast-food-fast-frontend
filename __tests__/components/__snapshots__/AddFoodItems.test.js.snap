// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddFoodItems Component should match snapshot 1`] = `
<section>
  <div
    class="container"
  >
    <div
      id="wrapper"
    >
      <h1>
        Enter Food Item details
      </h1>
      <div
        class="register"
      >
        <p>
          <input
            class="reg-control"
            name=""
            placeholder="Food Name"
            type="text"
          />
        </p>
        <p>
          <input
            class="reg-control"
            name=""
            placeholder="price"
            type="number"
          />
        </p>
        <p>
          <input
            class="reg-control"
            name=""
            placeholder="Quantity/portions Available"
            type="number"
          />
        </p>
        <p>
          <textarea
            cols="42"
            name=""
            placeholder="Description"
            rows="5"
          />
        </p>
        <p>
          <input
            accept="image/*"
            class="reg-control"
            name="pic"
            type="file"
          />
        </p>
        <p>
          <input
            class="button"
            name=""
            type="button"
            value="Submit"
          />
        </p>
      </div>
    </div>
  </div>
</section>
`;
